tests/containers/bci_prepare.pm

```diff
@@ -122,8 +122,8 @@ sub run {
 
     record_info('Clone', "Clone BCI tests repository: $bci_tests_repo");
     my $branch = $bci_tests_branch ? "-b $bci_tests_branch" : '';
-    script_run('rm -rf /root/BCI-tests');
-    assert_script_run("git clone $branch -q --depth 1 $bci_tests_repo /root/BCI-tests");
+    # script_run('rm -rf /root/BCI-tests');
+    assert_script_run("git clone $branch -q --depth 1 $bci_tests_repo /root/BCI-tests || (cd BCI-Tests && git fetch && git reset --hard origin)");
```

Comment:
Since this is executed now in create_hdd jobs, I would keep the `git clone` command here, and then, add this complex command in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/bci_test.pm#L92

Reply:
Changes done and working